### PR TITLE
Pull in new version of @bbc/psammead-copyright

### LINKED ISSUE #1788: Pull in new version of psammead-copyright
**Is your feature request related to a problem? Please describe.**
Pending completion and merge of https://github.com/bbc/psammead/issues/552, we should update Simorgh to pull in these changes, so that we can ensure the copyright component can be positioned correctly depending on service.

**Describe the solution you'd like**
Pull in the latest version of `psammead-copyright`, and write some logic to tell the component where it should be positioned within the `Figure` container.

1) add `dir` field to the Service Configs for News/Persian, etc. They should have values `ltr` / `rtl`.
2) use the value in the Copyright Container. 
3) pass in prop 'right' to Copyright component if text direction `rtl`, and 'left' if text direction is `ltr`. 

Note: that if the image copyright is owned by the BBC, we have logic to not include the copyright component at all.  

We should have Storybook examples for all variants. 

**Testing notes**
Checking Storybook examples as well as full pages: 
http://localhost:7080/news/articles/cn7769kpk9mo
http://localhost:7080/persian/articles/cwv2xv848j5o

Dev insight: Unit tests, (snapshot tests) and storybook.

### PR DESCRIPTION
**Pull request by 12**
Resolves #1788 

**Overall change:** Pull in `@bbc/psammead-copyright@1.0.0` to allow us to specify the position of the copyright component. Update service configs to define text direction.

**Code changes:**

- Pass `position` prop to `psammead-copyright` based on whether service is `ltr` or `rtl`.
- Define text direction in service configs
- Update snapshots

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

File: src/app/lib/config/services/persian.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ const persian = {
   serviceLocalizedName: 'فارسی',
   defaultImage: 'https://news.files.bbci.co.uk/ws/img/logos/og/persian.png',
   defaultImageAltText: 'BBC News فارسی',
+  dir: 'rtl',
```
Comment (dr3) [inline src/app/lib/config/services/persian.js:14]: Soz to be a pain, can you add `dir` to all services please?

**Review by dr3 — CHANGES_REQUESTED** [review]

File: src/app/containers/Copyright/index.test.jsx [hunk 2]
```diff
@@ -13,6 +13,7 @@ const newsServiceContextStub = {
   imageCopyrightOffscreenText: 'Image source, ',
```
Comment (sareh) [inline src/app/containers/Copyright/index.test.jsx:15]: In addition to Drew's comment -  adding `dir: 'ltr'` in the stubs for the LTR unit test cases.

**Review by sareh — APPROVED** [review]
Looks great. 
left right swap

**Review by dr3 — APPROVED** [review]

**Comment by jamesbrumpton** [comment]
LGTM.

**Pull request merged**